tests/commands/train_test.py

```diff
@@ -163,10 +164,19 @@ def test_other_modules(self):
             main()
 
         # Now add the --include-package flag and it should work.
-        sys.argv.extend(["--include-package", 'testpackage'])
+        # We also need to add --continue since the output directory already exists.
+        sys.argv.extend(["--continue", "--include-package", 'testpackage'])
 
         main()
 
+        # Rewrite out config file, but change a value
```

Comment:
Periods here and below.  (And if we can get pylint to check this automatically, I'm all for it - I'm pretty sure that Google's linter would catch these.)

Reply:
To be clear, I think it's a poor use of time for us to enforce capitalization or period in line comments.  I'm certainly not motivated to make automation for this, but I mentioned it because I think it's a particular poor use of time to flag this in code review.

That said, I'm willing to follow this if the team believes in it.  But old habits die hard, so I'm sure I'll need plenty of reminders.

Reply:
This PR has a clear example for why this is important, just 5 lines above this.  It's pretty likely that you would not have added a period on line 166 when you added a comment on line 167, and then the two-line comment block would be pretty hard to read.

I think consistent style is important enough that it's worth calling out in code review.  Better if we can automate it, but it took me about 10 seconds to add the comment (less if I hadn't added the line about pylint).